--- qemu-1.2.0/Makefile.orig	2013-01-16 14:56:09.124307697 -0500
+++ qemu-1.2.0/Makefile	2013-01-16 14:56:20.788370342 -0500
@@ -35,7 +35,7 @@
 
 $(call set-vpath, $(SRC_PATH))
 
-LIBS+=-lz $(LIBS_TOOLS)
+LIBS+=-lz -lrt $(LIBS_TOOLS)
 
 HELPERS-$(CONFIG_LINUX) = qemu-bridge-helper$(EXESUF)
 
@@ -172,7 +172,7 @@
 qemu-img-cmds.h: $(SRC_PATH)/qemu-img-cmds.hx
 	$(call quiet-command,sh $(SRC_PATH)/scripts/hxtool -h < $< > $@,"  GEN   $@")
 
-qemu-ga$(EXESUF): LIBS = $(LIBS_QGA)
+qemu-ga$(EXESUF): LIBS = -lrt $(LIBS_QGA)
 qemu-ga$(EXESUF): QEMU_CFLAGS += -I qga/qapi-generated
 
 gen-out-type = $(subst .,-,$(suffix $@))
--- qemu-1.2.0/Makefile.target.orig	2013-01-16 15:05:45.803313990 -0500
+++ qemu-1.2.0/Makefile.target	2013-01-16 15:06:12.934451375 -0500
@@ -34,6 +34,8 @@
 endif
 STPFILES=
 
+LIBS+=-lrt
+
 ifndef CONFIG_HAIKU
 LIBS+=-lm
 endif
