--- qemu-1.2.0/Makefile.target.orig	2013-01-16 14:48:05.492591233 -0500
+++ qemu-1.2.0/Makefile.target	2013-01-16 14:48:27.044717406 -0500
@@ -130,7 +130,7 @@
 obj-$(CONFIG_HAVE_CORE_DUMP) += dump.o
 obj-$(CONFIG_NO_GET_MEMORY_MAPPING) += memory_mapping-stub.o
 obj-$(CONFIG_NO_CORE_DUMP) += dump-stub.o
-LIBS+=-lz
+LIBS+=-lz -lrt
 
 QEMU_CFLAGS += $(VNC_TLS_CFLAGS)
 QEMU_CFLAGS += $(VNC_SASL_CFLAGS)
--- qemu-1.2.0/Makefile.orig	2013-01-16 14:56:09.124307697 -0500
+++ qemu-1.2.0/Makefile	2013-01-16 14:56:20.788370342 -0500
@@ -35,7 +35,7 @@
 
 $(call set-vpath, $(SRC_PATH))
 
-LIBS+=-lz $(LIBS_TOOLS)
+LIBS+=-lz -lrt $(LIBS_TOOLS)
 
 HELPERS-$(CONFIG_LINUX) = qemu-bridge-helper$(EXESUF)
 
