--- a/configure	2013-10-15 21:16:42.699382432 +0000
+++ b/configure	2013-10-15 21:16:33.308061324 +0000
@@ -4129,6 +4129,7 @@
 echo "HOST_CC=$host_cc" >> $config_host_mak
 echo "OBJCC=$objcc" >> $config_host_mak
 echo "AR=$ar" >> $config_host_mak
+echo "ARFLAGS=-r" >> $config_host_mak
 echo "AS=$as" >> $config_host_mak
 echo "CPP=$cpp" >> $config_host_mak
 echo "OBJCOPY=$objcopy" >> $config_host_mak
