--- configure.pulse	2011-08-03 12:25:20.000000000 +0200
+++ configure	2011-08-03 12:25:35.000000000 +0200
@@ -1532,7 +1532,7 @@
 
     pa)
     audio_drv_probe $drv pulse/simple.h "-lpulse-simple -lpulse" \
-        "pa_simple *s = NULL; pa_simple_free(s); return 0;"
+        "pa_simple *s = 0; pa_simple_free(s); return 0;"
     libs_softmmu="-lpulse -lpulse-simple $libs_softmmu"
     audio_pt_int="yes"
     ;;
